### PR TITLE
Edit FAQ

### PR DESCRIPTION
**Pull request by willingc**
Following up on the recent additions by @jtyberg (thanks!) in PR #2 , I have attempted to add more concrete wording for some of the pronoun references. Minimizing pronouns helps the reader avoid confusion.

@jtyberg Would you mind reviewing to make sure that I haven't changed your intended meaning or introduced an error into the FAQ with my changes? Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,20 +217,33 @@ docker logs jupyterhub
...
 
-In this deployment, JupyterHub uses DockerSpawner to spawn single-user Notebook servers.  The `jupyterhub_config.py` reads the image name from the `DOCKER_CONTAINER_IMAGE` environment variable.
+In this deployment, JupyterHub uses DockerSpawner to spawn single-user
+Notebook servers. You set the desired Notebook server image in a
+`DOCKER_CONTAINER_IMAGE` environment variable.
+
+When spawning single-user notebooks, JupyterHub reads the DockerSpawner's 
+image name from `jupyterhub_config.py` file. This file includes the
+DockerSpawner's image name which refers to desired image in the
+`DOCKER_CONTAINER_IMAGE` environment variable:
```
Comment (jtyberg) [inline README.md]: I get tripped up a bit on this one.  It reads like the `jupyterhub_config.py` is consulted when an image is spawned, but the `DOCKER_CONTAINER_IMAGE` is only read during JHub startup.  And one could interpret "DockerSpawner's image name" as the spawner itself having an image, but we're really talking about the Notebook server image that DockerSpawner uses to spawn a container.  

Perhaps: "JupyterHub reads the Notebook image name from `jupyterhub_config.py`.  This file retrieves the Notebook image name from the `DOCKER_CONTAINER_IMAGE` environment variable:"

I think the name of the variable may be causing some confusion.  Perhaps we should name it something like, `DOCKER_NOTEBOOK_IMAGE`, or something.

**Comment by jtyberg** [comment]
@willingc Cool! English has always been my third or fourth language 😉 

I left one comment.   Everything else looks good.

File: README.md (hunk 1 above)
Comment (willingc) [inline README.md]: > think the name of the variable may be causing some confusion. Perhaps we should name it something like, DOCKER_NOTEBOOK_IMAGE, or something.

I think that would be a good naming improvement.@minrk Thoughts?

Comment (willingc) [inline README.md]: As an FYI, there are 15 instances of DOCKER_CONTAINER_IMAGE across 5 files.

Comment (minrk) [inline README.md]: 👍 to DOCKER_NOTEBOOK_IMAGE

**Comment by jtyberg** [comment]
Merging this so I can do the DOCKER_NOTEBOOK_IMAGE rename.  We'll finalize wording in README in that PR.

**Comment by willingc** [comment]
Thanks @jtyberg. Sorry for the delay in responding - traveling.

**Pull request merged**